### PR TITLE
Analyzers show temperature in Kelvin as well as Celsius now

### PR DESCRIPTION
**Pull request by MrDoomBringer**
No reason why a scientific gas measurement device shouldn't be able to display its information in more than one unit.
image

:cl: MrDoomBringer
add: Analyzers now show temperature in Kelvin as well as Celsuis
/:cl:

[why]: 
image
image
image
image

alt title: how many commits does it take to screw in a lightblub?

### CODE REVIEW

**Comment by SpaceManiac** [comment]
>can't add/subtract 273 mentally

atmos jobban he

**Comment by MrDoomBringer** [comment]
@SpaceManiac listen

**Comment by as334** [comment]
This is massive powercreep disguised as a quality of life change, and if this gets merged the server will be so hideously unbalanced as to not be worth playing anymore.

**Comment by Cruix** [comment]
If it is scientific, why does it round to the nearest degree? It needs to display at least 12 significant figures.

**Comment by praisenarsie** [comment]
do fahrenheit too so the americans won't weep

**Comment by 81Denton** [comment]
@as334 wtf Bawhoppen why did you change your git name

**Comment by nicbn** [comment]
Fahrenheit is a joke

File: code/game/objects/items/devices/scanners.dm [hunk 1]
```diff
@@ -418,7 +418,7 @@ GAS ANALYZER
 				continue
 			var/gas_concentration = env_gases[id][MOLES]/total_moles
 			to_chat(user, "<span class='alert'>[env_gases[id][GAS_META][META_GAS_NAME]]: [round(gas_concentration*100, 0.01)] %</span>")
-		to_chat(user, "<span class='info'>Temperature: [round(environment.temperature-T0C)] &deg;C</span>")
+		to_chat(user, "<span class='info'>Temperature: [round(environment.temperature-T0C)] &deg;C ([round(environment.temperature)] &deg;K)</span>")
```
[the remark this answers is not in the document]
Comment (MrDoomBringer) [reply code/game/objects/items/devices/scanners.dm]: never realized this, huh

**Review by ninjanomnom — APPROVED** [review]
Sure why not

**Pull request merged**